coalib/results/Diff.py

```diff
@@ -200,9 +225,12 @@ def unified_diff(self):
         for equality comparison.
         """
         return ''.join(difflib.unified_diff(
-            self.original,
-            self.modified,
-            tofile=self.rename if isinstance(self.rename, str) else ''))
+                       self.original,
+                       self.modified,
+                       tofile=self.rename if isinstance(self.rename, str)
+                       else self.create
+                       if isinstance(self.create, str)
+                       else ''))
```

Comment:
I tried something like this 
```
+                       tofile=self.rename if isinstance(self.rename, str)
+                                          else self.create
+                                               if isinstance(self.create, str)
+                                               else ''))
```

but coala complained, I guess it was either pep8 or pycodestyle, you got a better suggestion :) ?